package.json

```diff
@@ -32,28 +32,29 @@
   "author": "algoan",
   "license": "ISC",
   "devDependencies": {
+    "@algoan/eslint-config": "^1.0.4",
     "@ava/typescript": "^1.1.1",
     "@commitlint/cli": "^8.2.0",
     "@commitlint/config-conventional": "^8.2.0",
     "@semantic-release/git": "^9.0.0",
-    "@types/pino": "^6.0.0",
+    "@types/pino": "^6.0.1",
     "@yelloan/tslint": "0.3.0",
-    "ava": "^3.8.1",
-    "commitizen": "^4.0.4",
-    "cz-conventional-changelog": "^3.0.2",
+    "ava": "^3.8.2",
+    "commitizen": "^4.1.2",
+    "cz-conventional-changelog": "^3.2.0",
+    "eslint": "^7.2.0",
     "google-pubsub-emulator": "^3.0.2",
     "husky": "^4.2.3",
-    "lint-staged": "^10.2.0",
-    "nyc": "15.0.1",
+    "lint-staged": "^10.2.9",
+    "nyc": "^15.1.0",
     "prettier": "^2.0.5",
-    "semantic-release": "^17.0.7",
+    "semantic-release": "^17.0.8",
     "tslint": "^6.1.2",
```

Comment:
Shouldn't you remove tslint?

Reply:
No it is a peer dependency of the [typescript-eslint/eslint-plugin-tslint](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin-tslint/package.json#L40) module